File: python/1159.py
def soma (n):
    cont = 0
    for i in range(0, 10):
        if n%2==0:
            cont+=n
        n+=1
    return cont


if __name__:
    n = int(input())
    while (n!=0):
        print(soma(n))
        n = int(input())

File: js/1159.js
var input = require('fs').readFileSync('./dev/stdin', 'utf8')
var lines = input.split('\n')

lines = lines.map(x => parseInt(x))


function soma(n) {

    let cont = 0
    for (let i =0;i<10;i++){
        if (n%2==0){
            cont+=n;
        }
        n++;
    }
    return cont;
}

while(1){
    n = lines.shift()
    if (n==0){
        break
    }
    console.log(soma(n)+'')
}